README.md

```diff
@@ -8,11 +10,46 @@
 
 Node implementation for BOA CoinNet
 
+# Docker usage
+
+We provide a public build of this repository (see above).
+The easiest way to get agora is to run `docker pull bpfk/agora`.
+
+The `Dockerfile` lives at the root of this repository,
+so one can run `docker build -t agora .` to build it.
+Note that you need to initialize submodules (`git submodules update --init`)
+before you first build agora.
+
+For a test run, try:
+```console
+docker run -p 127.0.0.1:4000:2826/tcp -v `pwd`/doc/:/root/etc/ agora -c etc/config.example.yaml
+```
+This will start a node with the example config file,
+and make the port locally accessible (See http://127.0.0.1:4000/) .
```

Comment:
Note: This currently gives a 404. We should probably make it more meaningful (and bind the administrative interface)